elasticdl/python/tests/optimizer_wrapper_test.py

```diff
@@ -14,6 +21,143 @@
 from elasticdl.python.tests.mock_kv_store import MockKvStore
 
 
+def _prepare_random_data(
+    iters_per_epoch,
+    batch_size,
+    input_length,
+    input_dim,
+    is_sparse,
+    random_seed,
+):
+    """
+    Generate data for training. `is_sparse=False` means we require that all
+    embedding ids appear in every batch data.
+    """
+    random.seed(random_seed)
+    X, Y = [], []
```

Comment:
Maybe use "features" and "labels" instead?

Reply:
"features" and "labels" are used for minibatch data in this test, i.e. 

```
for features, labels in zip(X, Y)
```